### ISSUE TITLE
Validation

### ISSUE THREAD

**Issue by r3code** [JonathanMataloni/uuid-v4#1]
4 UUID  has format  xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx
where x is one of hex digits 0-9A-F 
and y is one of 8, 9, A or B.

^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$

**Comment by jonamat** [comment]
Yeah it validates only the shape of the string but not all the content. We should update `validationRegex` and the `generate` method. Feel free to contribute! I'll bump the package version on npm
